include/small/ibuf.h

```diff
@@ -49,7 +49,7 @@ struct slab_cache;
  * Typical use case:
  *
  * struct ibuf *in;
- * coio_bread(coio, in, request_len);
+ * coio_read(coio, in, request_len);
```

Comment:
This isn't a typo. We actually have [`coio_bread`](https://github.com/tarantool/tarantool/blob/15b823f0ee3eb142c36564fa3973993c1ceba5b9/src/lib/core/coio_buf.h#L47) function in Tarantool.